### ISSUE TITLE
[crud-request] - Set filter conditions as array for RequestQueryBuilder?

### ISSUE THREAD

**Issue by chriszrc** [nestjsx/crud#131]
For the RequestQueryBuilder, we can pass in select fields as an array:

```
const queryString = RequestQueryBuilder
  .create()
  .select(['foo', 'bar'])
  .setFilter({ field: 'foo', operator: CondOperator.NOT_NULL })
  .setFilter({ field: 'bar', operator: CondOperator.NOT_NULL })
```

but filters have to be chained method calls. Was there an architectural reason for this? If not, this might actually be a PR I can make

**Comment by zMotivat0r** [comment]
Actually yeah, there is a code that I will push soon - it will add a method for passing an object of the query params so you'll be free to get rid of chaining. But if you think that you wanna create a PR, I'll be more than happy with that :)

**Comment by chriszrc** [comment]
Nice! Sorry, you lost me alittle, you already have the working code, but
you'd still want a PR?

**Comment by zMotivat0r** [comment]
@chriszrc I've added a possibility to add params on `.create()` (https://github.com/nestjsx/crud/wiki/Requests#usage). Hope that helps. Closing this issue for now.

**Comment by chriszrc** [comment]
Nice! Works great. I did notice that it appears switching to this method that the global config I had setup was no longer respected:

```
RequestQueryBuilder.setOptions({
       paramNamesMap: {
         filter: ['filter'],
       },
     });
```

But, I only included that originally because I think I was in between versions, and the backend wasn't accepting filter requests with the "filter[]" param, only "filter", but now I updated everything, and it's all working, so I can just remove this config. But I guess this should still work for users who want to override the param names?
